src/components/Footer.tsx

```diff
@@ -66,30 +62,17 @@ const sections = [
       },
     ],
   },
-  {
```

Comment:
A couple things happening here:

* All the BOS tech still works, but we've stopped using the "BOS" branding.  The idea of BOS is the same as the idea of Chain Abstraction, and since that's the term the industry is more accustomed to, that's what we're using instead.
* We didn't see many people using near.org as a directory of reusable components or to find other BOS gateways.  But people do like to see what applications are built on NEAR (whether BOS gateways or other apps), so that's what we want to highlight.